cpu/BUILD

```diff
@@ -59,6 +77,7 @@ constraint_value(
 constraint_value(
     name = "armv8-r",
     constraint_setting = ":cpu",
+    deprecation = "Do not use. Use a specific cpu product",
```

Comment:
Maybe you can just remove this altogether. There hasn't been a bazel_platforms release with this, right?

Reply:
+1, if it was never released we should be able to just delete it.

Reply:
Removed